zeppelin-web/src/index.html

```diff
@@ -132,6 +134,10 @@
     <script src="bower_components/bootstrap3-dialog/dist/js/bootstrap-dialog.min.js"></script>
     <script src="bower_components/floatThead/dist/jquery.floatThead.js"></script>
     <script src="bower_components/floatThead/dist/jquery.floatThead.min.js"></script>
+    <script src="bower_components/zeroclipboard/dist/ZeroClipboard.js"></script>
+    <script src="bower_components/moment/moment.js"></script>
+    <script src="bower_components/pikaday/pikaday.js"></script>
+    <script src="bower_components/handsontable/dist/handsontable.js"></script>
```

Comment:
ZeroClipboard and pikaday.js are not required, need to see how to remove them from the build.
As of now moment.js is used only for number formatting (7bb508e)